mailing_list/static/mailing_list/js/app.js

```diff
@@ -77,6 +79,14 @@
             ml.loaded = true;
         });
 
+        $http.get(GET_COURSE).success(function(data){
+            ml.course_code = data.course_code;
+        });
```

Comment:
Any way to bake the course_code into the mailing list data to avoid this extra call?